ios-target/View/Home/MatchNotificationViewController.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  MatchNotificationViewController.swift
+//  ios-target
+//
+//  Created by sol manini on 7/3/19.
+//  Copyright © 2019 TopTier labs. All rights reserved.
+//
+
+import UIKit
+
+class MatchNotificationViewController: ModalViewController {
+  
+  @IBOutlet weak var connectMatchButton: UIButton!
+  @IBOutlet weak var userNameLabel: UILabel!
+  @IBOutlet weak var userImage: UIImageView!
+  
+  var viewModel = MatchNotificationViewModel()
+  weak var delegate: MatchNotificationViewModelDelegate?
+  
+  override func viewDidLoad() {
+    super.viewDidLoad()
+    connectMatchButton.backgroundColor = .macaroniAndCheese
+    if let userEmail = viewModel.user?.email {
```

Comment:
the text property of UILabel is optional so you don't need to check before assigning it.